### DISCUSSION TITLE
Credit disappeared

### DISCUSSION THREAD

**Discussion by ghbaud** [deepgram/community#872] [General help] [answered]
I had to put my project on hold for a couple of months, but now that I'm working on it again I noticed that most of the approximately $300 in credits that I had on my account have disappeared. Could those missing credits be restored please?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment]
@ghbaud It's possible those credits expired based on: your sign-up date OR purchase date AND what type of credits there were.

* Pay As You Go credit never expires. 
* Growth plan credit expire 1 year from purchase unless you renew or upgrade.
* Free Credits expire 1 year after sign up.

If you can let me know what your `project_id` I can take a closer look to see what might be going on here.

**Comment by ghbaud** [comment]
Thanks for looking into it. My project ID is 7c61a551-6ab4-42d9-a270-fea13eff7b40

**Comment by jpvajda** [comment]
@ghbaud  From what I can tell you had a block of `$193.07` credits expire on July 3rd 2024, and you still have a set of promotional credits of `$100.00` valid to March 14, 2025.  I'm looking into this a bit further.  Do you know if that July 3rd block was relating to when you signed up for Deepgram?

**Reply by ghbaud** [reply]
I think the $193 block was what was left of a $200 credit I got from a deepgram questionnaire I filled out about what I was working on, how I was using it, and so on. I didn't know it had an expiration date.

**Reply by jpvajda** [reply, marked as answer]
@ghbaud  got it. Most of our promotional credits do expire after a year, I'm going to re-allocate these for you.

**Comment by ghbaud** [comment]
Thank you! Very kind of you.
